docs/getting_started/software_available.md

```diff
@@ -12,28 +20,33 @@ We have installed IGV to run from the linux Terminal command line (not the web b
 
 Example for the 5236 Wellcome Sanger Institute low depth exomes:
 
-/usr/local/bin/IGV\_Linux\_2.12.3/igv.sh --genome /genesandhealth/library-red/genesandhealth/exome\_seq/2019\_11\_\_5236\_GNHonly/crams/hs38DH.fa /genesandhealth/library-red/genesandhealth/exome\_seq/2019\_11\_\_5236\_GNHonly/crams/sc\_autozygELGH6823965.cram
+`/usr/local/bin/IGV\_Linux\_2.12.3/igv.sh --genome /genesandhealth/library-red/genesandhealth/exome\_seq/2019\_11\_\_5236\_GNHonly/crams/hs38DH.fa /genesandhealth/library-red/genesandhealth/exome\_seq/2019\_11\_\_5236\_GNHonly/crams/sc\_autozygELGH6823965.cram`
 
 The default set of (hg19) files for IGV are found here
 
-/genesandhealth/library-green/sanger/igv/
+`/genesandhealth/library-green/sanger/igv/`
 
 ### Jupyter notebooks
 
-Is available in Applications menu as graphical user interface (not from linux command line).
+Jupyter is available in Applications menu as graphical user interface and can be run from the linux command line, by running `jupyter notebook` in the terminal. 
+
+## VS Code
+
+The VS Code editor is available in Applications menu as graphical user interface (not from linux command line). It does not have plugins available unless they are specifically requested and installed by the Sanger Institute.
```

Comment:
The VS Code has certain extensions installed (we can add a list of available extensions). We can also mention the Jupyter Notebooks extension is already installed on VS Code.